src/glmfit.jl

```diff
@@ -475,6 +480,7 @@ function fit(::Type{M},
     y::AbstractVector{<:Real},
     d::UnivariateDistribution,
     l::Link = canonicallink(d);
+    dropcollinear::Bool = false,
```

Comment:
For consistency with `lm` this should be `true` by default. Then we should also test that passing `dropcollinear=false` throws an error when columns are collinear.

Reply:
Okay. We also need to update existing examples in `glm`  where it prints the model.